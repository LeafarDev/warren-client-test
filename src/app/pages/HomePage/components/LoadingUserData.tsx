import React from 'react';
import { CircularProgress, Grid } from '@material-ui/core';

export function LoadingUserData(): JSX.Element {
  return (
    <Grid
      container
      spacing={0}
      direction="column"
      alignItems="center"
      justify="center"
      style={{ minHeight: '100vh' }}
    >
      <CircularProgress disableShrink/>

    </Grid>
  );
}
