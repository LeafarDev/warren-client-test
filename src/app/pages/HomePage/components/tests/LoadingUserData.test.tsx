import React from 'react';
import { LoadingUserData } from '../LoadingUserData';
import { render } from '@testing-library/react';

describe('<LoadingUserData />', () => {

  it('should render and match the snapshot', () => {
    const {
      container: { firstChild }
    } = render(<LoadingUserData/>);

    expect(firstChild)
      .toMatchSnapshot();
  });
});
