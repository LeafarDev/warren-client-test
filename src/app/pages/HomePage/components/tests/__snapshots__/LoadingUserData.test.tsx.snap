// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<LoadingUserData /> should render and match the snapshot 1`] = `
<div
  class="MuiGrid-root MuiGrid-container MuiGrid-direction-xs-column MuiGrid-align-items-xs-center MuiGrid-justify-xs-center"
  style="min-height: 100vh;"
>
  <div
    class="MuiCircularProgress-root MuiCircularProgress-colorPrimary MuiCircularProgress-indeterminate"
    role="progressbar"
    style="width: 40px; height: 40px;"
  >
    <svg
      class="MuiCircularProgress-svg"
      viewBox="22 22 44 44"
    >
      <circle
        class="MuiCircularProgress-circle MuiCircularProgress-circleDisableShrink MuiCircularProgress-circleIndeterminate"
        cx="44"
        cy="44"
        fill="none"
        r="20.2"
        stroke-width="3.6"
      />
    </svg>
  </div>
</div>
`;
